debian/nntrainer.install

```diff
@@ -0,0 +1 @@
+/usr/lib/*/*.so
```

Comment:
If you want general applications to use nntrainer, the public .so files should be in libdir, not lib/*/...

Reply:
Uh.. sorry. I thought you were talking about /usr/lib/nntrainer/libnntrainer.so
If that was /usr/lib/x86_64.../libnntrainer.so (MULTIARCH), that's fine.

Reply:
Fixed.